cogs/smallfunctions.py

```diff
@@ -0,0 +1,43 @@
+import discord
+import random
+import #new file storing priv variables
+from discord.ext import commands
+
+# Class responsible for all the small commands run by server members.
+class SmallFunctions(commands.cog):
+
```

Comment:
Add in the init function here to initialize the class. Try to follow what the other functions are doing. Here's how Python's init constructors work: https://www.geeksforgeeks.org/__init__-in-python/#